Dockerfile

```diff
@@ -36,10 +38,10 @@ RUN apt update -qq && apt install -qq -y --no-install-recommends \
         libgl1 \
         libtcmalloc-minimal4 \
         make \
-        openjdk-8-jdk-headless \
         openjdk-11-jdk-headless \
         openssh-client \
         patch \
+        python2 \
```

Comment:
why python2 needed here ?

Reply:
Because Buck is built using Python2: https://buck.build/setup/getting_started.html